l10n_it_vat_statement_split_payment/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Silvio Gregorini (silviogregorini@openforce.it)
+# Copyright (c) 2018 Openforce Srls Unipersonale (www.openforce.it)
+# Copyright (c) 2019 Matteo Bilotta
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': "ITA - Liquidazione IVA + Scissione dei pagamenti",
+    'summary': """Improvements to end period VAT statements"""
+               """ through implementation of split payments""",
```

Comment:
Visto che non è traducibile andrebbe in italiano.

Reply:
Fatto!  
Grazie! 😉